### PR TITLE
feat: index carbon components - v10

### LINKED ISSUE #515: Index Carbon Components

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #515 

Indexing PR: https://github.com/carbon-design-system/carbon/pull/11265

Testing / reviewing
Run the app locally you should see `carbon components` listed as a library and the components should render (44 total) with the filters as shown below:

Screen Shot 2022-04-21 at 3 42 49 PM

### CODE REVIEW

File: services/web-app/data/libraries.js
Comment (francinelucca) [inline services/web-app/data/libraries.js:400]: do we still want to go with sponsor: carbon given our discussions today regarding carbon-components? CC: @mattrosno

Comment (mattrosno) [reply services/web-app/data/libraries.js:400]: The Carbon core team does *not* sponsor the vanilla JS components. They have been community maintained for the last couple years.

Comment (francinelucca) [reply services/web-app/data/libraries.js:400]: I think we can just remove the key then @andreancardona

Comment (francinelucca) [inline services/web-app/data/libraries.js:400]:
```suggestion
  'carbon-components': {
    host: 'github.com',
    org: 'carbon-design-system',
    repo: 'carbon',
    path: '/packages/carbon-components',
```

File: services/web-app/data/libraries.js
Comment (francinelucca) [inline services/web-app/data/libraries.js:156]:
```suggestion
  //   host: 'github.com',
  //   org: 'carbon-design-system',
  //   repo: 'carbon',
  //   path: '/packages/carbon-components',
  //   group: carbonComponentsGroup
```

**Review by francinelucca — CHANGES_REQUESTED** [review]

File: services/web-app/data/libraries.js
Comment (mattrosno) [inline services/web-app/data/libraries.js:151]: For this, the path should be `/packages/components` and the ref should be `v10` and that should allow us to index this library in the v10 branch instead of the default branch (main).

File: services/web-app/data/libraries.js
Comment (mattrosno) [inline services/web-app/data/libraries.js:401]: Update path and ref to your new PR.

**Review by mattrosno — CHANGES_REQUESTED** [review]
### Copy button

I don't see any links in copy button: http://localhost:3000/assets/carbon-components/latest/copy-button

There should be a demo: https://the-carbon-components.netlify.app/?nav=copy-button

image

### Data table

http://localhost:3000/assets/carbon-components/latest/data-table

We sure this should be tagged "input control"?

image

### Grid

The grid's demo incorrectly takes you here: https://the-carbon-components.netlify.app/?nav=form

### Toolbar

Missing thumbnail.

image

File: services/web-app/data/libraries.js
Comment (mattrosno) [inline services/web-app/data/libraries.js]:
```suggestion
    group: carbonComponentsGroup,
    ref: 'v10'
```

File: services/web-app/data/libraries.js
Comment (mattrosno) [inline services/web-app/data/libraries.js]:
```suggestion
    group: carbonComponentsGroup,
    ref: 'v10'
```

**Review by mattrosno — CHANGES_REQUESTED** [review]
Need v10 refs.

**Comment by mattrosno** [comment]
@andreancardona we can merge this if you open a separate issue to fix the copy button, data table, and grid links in the carbon monorepo v10 branch.

**Comment by andreancardona** [comment]
@mattrosno I added those `v10` refs - I'll take a look at the other items shortly.  

Thanks!

**Review by mattrosno — APPROVED** [review]

**Review by francinelucca — APPROVED** [review]

**Pull request merged**
